palace/drivers/eigensolver.hpp

```diff
@@ -40,13 +41,10 @@ class EigenSolver : public BaseSolver
                       int i, std::complex<double> omega, double Em) const;
 
 public:
-  EigenSolver(const IoData &iodata, bool root, int size = 0, int num_thread = 0,
-              const char *git_tag = nullptr)
-    : BaseSolver(iodata, root, size, num_thread, git_tag)
-  {
-  }
+  using BaseSolver::BaseSolver;
 
-  void Solve(std::vector<std::unique_ptr<mfem::ParMesh>> &mesh) const override;
+  ErrorIndicators
+  Solve(const std::vector<std::unique_ptr<mfem::ParMesh>> &mesh) const final;
```

Comment:
We don't use `final` anywhere else in the code, and it doesn't really serve a purpose here. Let's keep this as `override`.